client/state/action-types.ts

```diff
@@ -977,8 +977,6 @@ export const URL_ANALYZER_ANALYZE_SUCCESS = 'URL_ANALYZER_ANALYZE_SUCCESS';
 export const URL_ANALYZER_ANALYZE_ERROR = 'URL_ANALYZER_ANALYZE_ERROR';
 export const URL_ANALYZER_RESET_ERROR = 'URL_ANALYZER_RESET_ERROR';
 export const URL_ANALYZER_URL_DATA_UPDATE = 'URL_ANALYZER_URL_DATA_UPDATE';
-export const USER_DEVICES_ADD = 'USER_DEVICES_ADD';
-export const USER_DEVICES_REQUEST = 'USER_DEVICES_REQUEST';
 export const USER_LICENSES_RECEIVE = 'USER_LICENSES_RECEIVE';
 export const USER_LICENSES_REQUEST = 'USER_LICENSES_REQUEST';
 export const USER_LICENSES_REQUEST_SUCCESS = 'USER_LICENSES_REQUEST_SUCCESS';
```

Comment:
The `client/state/data-layer/wpcom/me/block/f` file looks like something we should remove from the branch. 

Sorry for commenting here, but GH won't let me comment inside an empty file for some reason 🤷‍♂️